File: Node/Infrastructure/FuncionarioRepository.js
const Funcionario = require('../Domain/Funcionario');
const sqlite3 = require('sqlite3').verbose();

module.exports = class FuncionarioRepository {
    GetAll() {
        let funcionarios = [];    
        let db = GetDb();

        db.all('SELECT Id, Nome, DataCadastro FROM Funcionarios', (erro, linhas) => {
            for(let i in linhas) {
                funcionarios.push(new Funcionario(linhas[i].Nome, linhas[i].Id, linhas[i].dataCadastro));
            }
        });        
        
        return funcionarios;
    }   
}

function GetDb() {
    return new sqlite3.Database('/Users/jorgepaulo/Documents/Códigos/Serpro/TreinoSerpro/Base/mydb.db');
}

File: Python/Infrastructure/Repositories/FuncionarioRepository.py
import sqlite3
from Python.Domain.Models.Funcionario import Funcionario

class FuncionarioRepository:
    
    def GetAll(self):
        funcionarios = []

        connection = self.GetConnection()
        for row in connection.execute('SELECT Id, Nome, DataCadastro FROM Funcionarios'):
            funcionarios.append(Funcionario(row[1], row[0], row[2]))
        
        return funcionarios

    def GetConnection(self):
        return sqlite3.connect('/Users/jorgepaulo/Documents/Códigos/Serpro/TreinoSerpro/Base/mydb.db')